client/my-sites/media-library/media-file.tsx

```diff
@@ -55,10 +54,9 @@ const MediaFile: React.FC< MediaFileProps > = function MediaFile( {
 export default connect( ( state: IAppState, { src }: Pick< MediaFileProps, 'src' > ) => {
 	const siteId = getSelectedSiteId( state );
 	const siteSlug = getSelectedSiteSlug( state ) as string;
-	const isAtomic = !! isSiteAutomatedTransfer( state, siteId as number );
-	const isPrivate = !! isPrivateSite( state, siteId ?? 0 );
 	const { filePath, query, isRelativeToSiteRoot } = mediaURLToProxyConfig( src, siteSlug );
-	const useProxy = Boolean( isAtomic && isPrivate && filePath && isRelativeToSiteRoot );
+	const isJetpack = isJetpackSite( state, siteId ?? 0, { treatAtomicAsJetpackSite: false } );
+	const useProxy = Boolean( ! isJetpack && filePath && isRelativeToSiteRoot );
```

Comment:
Not a huge deal, but I'd say a more idiomatic way of doing this and making sure the return type is always bool would be to:

cost isProxy = ! isJetpack && !!filePath && isRelativeToSiteRoot;

This makes sure the value for `filePath` is converted to bool, and makes the expression return `bool` as well, without the need to call `Boolean()` over it.

[Here's more info about the double-negation idiom](https://medium.com/@tariibaba/javascript-double-negation-e667b993e408).

Reply:
I also see that `isJetpack` is not properly typed, and might be `null`, but that's a problem in the `isJetpackSite` function, not in the cod here . Again, in practice, it will not matter as `null` would basically equal false(y) and the `!` will make sure to return a `bool` value.